app/editor/src/store/hooks/admin/useReports.ts

```diff
@@ -49,13 +49,16 @@ export const useReports = (): [IAdminState & { initialized: boolean }, IReportCo
       },
       getReport: async (id: number) => {
         const response = await dispatch<IReportModel>('get-report', () => api.getReport(id));
-        store.storeReports((reports) =>
-          reports.map((ds) => {
-            if (ds.id === response.data.id) return response.data;
-            return ds;
-          }),
-        );
-        return response.data;
+        if (response.status === 200) {
```

Comment:
We should be doing this more often.  When there is a failure our response has invalid data.